bmaptools/BmapHelpers.py

```diff
@@ -95,70 +93,61 @@ def program_is_available(name):
 
     return False
 
+def get_mount_point(path):
+    """Return the mount point for 'path'."""
 
-def get_mount_point(filepath):
-    """
-    Return the mount point the file object file resides on.
-    """
-    f = os.path.realpath(filepath)
-    while f != os.path.sep:
-        if os.path.ismount(f): return f
-        f = os.path.realpath(os.path.join(f, os.pardir))
-    return f
+    part = os.path.realpath(path)
+    while part != os.path.sep:
+        if os.path.ismount(part): 
+            return part
+        part = os.path.realpath(os.path.join(part, os.pardir))
+    return part
 
+def get_file_system_type(path):
```

Comment:
Ditto (private vs public function).

Reply:
given that this is a helpers module I fell this particular method could be quite useful going forward for any future work that may need to know the file system type of a path, even though its only currently used internally. If you would prefer it to be private I can change it.